src/Migrations/M011TurnIncludeArticlesSettingIntoBlock.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+use WP_Post;
+
+/**
+ * Add the Articles block as a block to all posts that use it with the "Include Articles In Post" settings field, and remove field.
+ */
+class M011TurnIncludeArticlesSettingIntoBlock extends MigrationScript {
+
+	private const INCLUDE_ARTICLES_META_KEY = 'include_articles';
+
+	/**
+	 * Perform the actual migration.
+	 *
+	 * @param MigrationRecord $record Information on the execution, can be used to add logs.
+	 *
+	 * @return void
+	 */
+	protected static function execute( MigrationRecord $record ): void {
+		$args = [
+			'posts_per_page' => - 1,
+			'post_type'      => 'post',
+			'meta_query'     => [
+				[
+					'key'     => self::INCLUDE_ARTICLES_META_KEY,
+					'value'   => '',
+					'compare' => 'NOT IN',
+				],
+			],
+		];
+
+		$article_posts = get_posts( $args );
+
+		echo 'Converting ' . count( $article_posts ) . " posts with \"Include Articles In Post\" setting to blocks.\n";
+
+		foreach ( $article_posts as $article_post ) {
+			self::append_articles_block_from_meta( $article_post );
+			delete_post_meta( $article_post->ID, self::INCLUDE_ARTICLES_META_KEY );
+		}
+	}
+
+	/**
+	 * Add the Articles block that is in the meta to the end of the post content.
+	 *
+	 * @param WP_Post $post A post that has a TAB set through its meta.
+	 */
+	private static function append_articles_block_from_meta( WP_Post $post ): void {
+		if ( 'yes' === $post->include_articles ) {
+			// Build the block code for the articles block.
+			$block_attributes = [
+				'exclude_post_id' => $post->ID,
```

Comment:
No need to pass a "exclude_post_id" param here, as the article block itself taking care of [adding this param](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/blocks/Articles/useArticlesFetch.js#L33) when fetch the articles, hence removing it.